### PR TITLE
build: add update split repo script to update the README and delete the rest of the repo

### PR DESCRIPTION
**Pull request by sofisl**
According to agreement with DEE org, since the split repo will stay un-archived during the migration, we'll need to delete everything but the samples, while updating the README.md. We'll also need to add or replace the owlbot.py entirely to ignore the README.

### CODE REVIEW

File: bin/update-split-repo.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  exit 1
+fi
+
+# repo name (e.g. nodejs-asset)
+SPLIT_REPO=$1
+# destination directory (e.g. google-cloud-asset)
+ARTIFACT_NAME=$2
+
+rm -rf /tmp/${SPLIT_REPO}
+git clone git@github.com:googleapis/${SPLIT_REPO}.git /tmp/${SPLIT_REPO}
```
Comment (chingor13) [inline bin/update-split-repo.sh:30]: Whenever doing string interpolation in bash, be sure to quote the values so you cannot inject commands.
```suggestion
git clone "git@github.com:googleapis/${SPLIT_REPO}.git" "/tmp/${SPLIT_REPO}"
```

File: bin/update-split-repo.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+git checkout -b 'removeOwlBotAndUpdateREADME'
+
+shopt -s extglob
+
+# Delete everything first that's not the samples, README.md, and git folder
+rm -rf !("samples"|"README.md"|".git") 
+rm -rf /tmp/${SPLIT_REPO}/.[!.git]*
+rm -rf /tmp/${SPLIT_REPO}/.github
+
+IGNORE_README_TXT="# Copyright 2021 Google LLC
```
Comment (chingor13) [inline bin/update-split-repo.sh:43]: 2022?

File: bin/migrate-split-repo.sh [hunk 3]
```diff
@@ -41,3 +41,6 @@ ${SCRIPT_DIR}/migrate-git-history.sh \
   "${PACKAGE_PATH}" \
```
Comment (chingor13) [inline bin/migrate-split-repo.sh:41]: It looks like this file's mode is changing to not be executable?

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: bin/migrate-split-repo.sh (hunk 3 above)
Comment (sofisl) [reply bin/migrate-split-repo.sh:41]: Fixed it, thanks!

**Comment by sofisl** [comment]
@chingor13 thanks for the review, PTAL

File: bin/update-split-repo.sh [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  echo "Usage: $0 <split-repo-name> <target-path>"
+  exit 1
+fi
+
+# repo name (e.g. nodejs-asset)
+SPLIT_REPO=$1
+# destination directory (e.g. google-cloud-asset)
+ARTIFACT_NAME=$2
+
+rm -rf /tmp/${SPLIT_REPO}
```
Comment (chingor13) [inline bin/update-split-repo.sh:29]: All of these string interpolations need quotes. I didn't mark all of them because there were so many.

**Review by chingor13 — CHANGES_REQUESTED** [review]

**Review by chingor13 — APPROVED** [review]

**Pull request merged**